roles/ceph-monitor/files/ceph_adjust.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# ceph wants pg number be power of 2(good for cursh).
+# So if we increase pg number from 1024, the next number will be 2048.
+# Increasing so many pg numbers will cause much data rebalancing.
+# This script is to increase pg number in small step,
+# we run this cript as cron job every 5 minutes.
+
+import subprocess
+import pickle
+import logging
+import json
+import os
+import time
+import re
+import glob
+
+PKLFILE = '/var/lib/ceph_adjust/pgsonpool.pkl'
+LOGFILE = '/var/log/ceph/ceph_adjust.log'
+
+handler = logging.FileHandler(LOGFILE)
```

Comment:
If we move this to its own python module, this logging logic may need to change.